api/src/client/ApiFormationClient.ts

```diff
@@ -226,8 +234,43 @@ export const ApiFormationClient = (config: ApiConfig, logger: LogWriterType): Fo
             },
           };
         }
+        case "nonprofit":
+        case "veteran-nonprofit": {
+          return {
+            AdditionalDomesticNonProfitCorp: {
+              HasMembers: formationFormData.hasNonprofitBoardMembers ? "Yes" : "No",
+              MemberTermsProvisionLocation:
+                formationFormData.nonprofitBoardMemberQualificationsSpecified === "IN_FORM"
+                  ? "Herein"
+                  : "Bylaw",
+              MemberTerms:
+                formationFormData.nonprofitBoardMemberQualificationsSpecified === "IN_FORM"
+                  ? formationFormData.nonprofitBoardMemberQualificationsTerms
+                  : undefined,
+              MemberClassPermissionsProvisionLocation:
+                formationFormData.nonprofitBoardMemberRightsSpecified === "IN_FORM" ? "Herein" : "Bylaw",
+              MemberClassPermissions:
+                formationFormData.nonprofitBoardMemberRightsSpecified === "IN_FORM"
+                  ? formationFormData.nonprofitBoardMemberRightsTerms
+                  : undefined,
+              TrusteeElectionProcessProvisionLocation:
+                formationFormData.nonprofitTrusteesMethodSpecified === "IN_FORM" ? "Herein" : "Bylaw",
+              TrusteeElectionProcess:
+                formationFormData.nonprofitTrusteesMethodSpecified === "IN_FORM"
+                  ? formationFormData.nonprofitTrusteesMethodTerms
+                  : undefined,
+              AssetDistributionProvisionLocation:
+                formationFormData.nonprofitAssetDistributionSpecified === "IN_FORM" ? "Herein" : "Bylaw",
+              AssetDistribution:
+                formationFormData.nonprofitAssetDistributionSpecified === "IN_FORM"
+                  ? formationFormData.nonprofitAssetDistributionTerms
+                  : undefined,
```

Comment:
[sand] a lot of copy-paste in this section. Could be refactored as a helper method instead of copying the same logic so many times?